packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/config_with_when.yaml

```diff
@@ -0,0 +1,24 @@
+framework:
+    test: false
+
+when@test:
+    framework:
+        test: true
+        session:
+            storage_factory_id: session.storage.factory.mock_file
+-----
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Config\FrameworkConfig;
+
+return static function (FrameworkConfig $framework, ContainerConfigurator $container) {
+    $framework->test(false);
+
+    if ('test' === $container->env()) {
+        $framework->test(true);
+        $framework->session()->storageFactoryId('session.storage.mock_file');
```

Comment:
@tacman how `storage_factory_id` become `storageFactoryId` ? I never use it before, is there a way to configure it via something like `->configure()` method, eg: ?

```php
$containerConfigurator->get('session')
    ->configure(['storage_factory_id' => 'session.storage.mock_file']);
```

Reply:
@samsonasik Yes, as this convers only YAML to PHP, not to PHP fluent configs. That's different operation.

Reply:
Possibly, the inner content is not really important as we already cover it. 
The feature is about the ` if ('test' === $container->env()) {` condition.

Reply:
ok, updated the fixture at https://github.com/symplify/symplify/pull/4179/commits/a84b2fab14f4a0c5e186add3bff82d16b4039a99